tests/test_runner/test_runner.py

```diff
@@ -676,6 +677,35 @@ def test_init(self):
                                     '`param_scheduler` should be a'):
             Runner(**cfg)
 
+        # 7. Test auto distinguish experiment
+        cur_dir = os.getcwd()
+        os.chdir(self.temp_dir)
+        self.temp_dir = 'work_dirs'
```

Comment:
We should not overwrite `self.temp_dir` here, otherwise `tearDown` cannot remove the temporary dirctory.